Src/ElasticScale.Client/Microsoft.Azure.SqlDatabase.ElasticScale.Client.csproj

```diff
@@ -7,7 +7,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <Version>2.4.1</Version>
     <Authors>Microsoft</Authors>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <PackageTags>Microsoft;Elastic;Scale;Azure;SQL;DB;Database;Shard;Sharding;Management;Query;azureofficial</PackageTags>
     <PackageReleaseNotes>Updated dependent nugets (due to CVEs), added support for ActiveDirectoryDefault Entra auth</PackageReleaseNotes>
```

Comment:
>Updated dependent nugets (due to CVEs), added support for ActiveDirectoryDefault Entra auth [](http://example.com/codeflow?start=25&length=91)

Update these release notes to indicate what is new

Reply:
Done.